Source/WPE/CMakeLists.txt

```diff
@@ -176,6 +184,22 @@ if (USE_WPE_BACKEND_WESTEROS)
     )
 endif ()
 
+if (USE_WPE_BACKEND_STM)
+     list(APPEND WPE_INCLUDE_DIRECTORIES
+        ${STM_INCLUDE_DIRS}
+        ${STMEGL_INCLUDE_DIRS}
+     )
+     list(APPEND WPE_LIBRARIES
+        ${STM_LIBRARIES}
+        ${STMEGL_LIBRARIES}
+     )
+     list(APPEND WPE_SOURCES
+        Source/Graphics/STM/RenderingBackendSTM.cpp
+        Source/Graphics/RenderingBackend.cpp
```

Comment:
I don't think you need this line, the file is already included in the list on line 114.